examples/bri-3/src/bri/workgroup/workgroups/models/workgroup.ts

```diff
@@ -76,4 +76,8 @@ export class Workgroup {
   public updateParticipants(newParticipants: BpiSubject[]): void {
     this.participants = newParticipants;
   }
+
+  public updateStatus(newStatus: WorkgroupStatus): void {
+    this.status = newStatus;
```

Comment:
if(status != Active)
 throw Internal Serve Error ; // as this should never be possible to happen
else ...

Reply:
resolved comment, but should be fixed too @biscuitdey